content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -7,118 +7,236 @@ title: Bring your own bucket (BYOB)
+    **Multi-tenant Cloud only**:
+    For team level storage in Multi-tenant Cloud, obtain your W&B organization ID, which is required in your bucket policy for team level storage:
+      1. Log in to W&B as a user with the `admin` role.
+      1. Click the icon at the top left to open the left navigation, then click **Create a team to collaborate**.
+      1. Click **External storage**, then set **Cloud Provider** to **CoreWeave**.
+      1. Near the bottom of the dialog, make a note of the **W&B organization ID**.
+      1. Keep this page open. Later, you will use it to create the new team.
+    
+    Replace:
+    - In `Resource`, each instance of `<cw_bucket>` with the CoreWeave bucket name.
+    - In `Principal` and `Condition`, each instance of `<wb-org-id>` with your W&B organization ID.
 
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Id": "WandBAccess",
+      "Statement": [
+        {
+          "Sid": "WAndBAccountAccess",
+          "Effect": "Allow",
+          "Principal": { "CW": "arn:aws:iam::<wb-org-id>:*" },
+            "Action" : [
+              "s3:GetObject*",
+              "s3:GetEncryptionConfiguration",
+              "s3:ListBucket",
+              "s3:ListBucketMultipartUploads",
+              "s3:ListBucketVersions",
+              "s3:AbortMultipartUpload",
+              "s3:DeleteObject",
+              "s3:PutObject",
+              "s3:GetBucketCORS",
+              "s3:GetBucketLocation",
+              "s3:GetBucketVersioning"
+            ],
+            "Resource": [
+              "arn:aws:s3:::<cw_bucket>",
+              "arn:aws:s3:::<cw_bucket>/*"
+            ],
+            "Condition": {
+			        "StringLike": {
+				        "wandb:OrgID": [
+					       "<wb-org-id>"
+				        ]
+			        }
+		        }
+        }
+      ]
+    }
+    ```
 
-## Cloud storage in same cloud as W&B platform
+    </details>-->
 
-Based on your use case, configure a storage bucket at the team or instance level. How a storage bucket is provisioned or configured is the same irrespective of the level it's configured at, except for the access mechanism in Azure. 
 
-{{% alert %}}
-W&B recommends that you use a Terraform module managed by W&B to provision a storage bucket along with the necessary access mechanism and related IAM permissions:
+Keep a record of the bucket name and access credentials. Next, [configure W&B](#configure-byob).
 
-* [AWS](https://github.com/wandb/terraform-aws-wandb/tree/main/modules/secure_storage_connector)
-* [GCP](https://github.com/wandb/terraform-google-wandb/tree/main/modules/secure_storage_connector)
-* Azure - [Instance level BYOB](https://github.com/wandb/terraform-azurerm-wandb/tree/main/examples/byob) or [Team level BYOB](https://github.com/wandb/terraform-azurerm-wandb/tree/main/modules/secure_storage_connector)
-{{% /alert %}}
+<details>
+<summary>Troubleshoot CoreWeave storage connection errors</summary>
+<h4>Troubleshoot CoreWeave storage connection errors</h4>
```

Comment:
Don't think we need this h4: 
<img width="814" alt="Screenshot 2025-06-27 at 1 23 09 PM" src="https://github.com/user-attachments/assets/a0d9b59c-8dfa-4458-89e5-29bed7279881" />

Reply:
Done (also moved the whole troubleshooting section to the end)